cryodrgn/commands/filter.py

```diff
@@ -80,13 +81,19 @@ def add_args(parser: argparse.ArgumentParser) -> None:
         help="path to a file containing previously selected indices "
         "that will be plotted at the beginning",
     )
+    parser.add_argument(
+        "--sel-dir",
+        type=str,
+        help="directory to save the particle selection into",
+    )
 
 
 def main(args: argparse.Namespace) -> None:
     workdir = args.outdir
     epoch = args.epoch
     kmeans = args.kmeans
     plot_inds = args.plot_inds
+    sel_dir = args.sel_dir
```

Comment:
we can just use `args.sel_dir` throughout (and should probably do the same for the other attributes defined above)